sdk/rust/libveracruz/src/lib.rs

```diff
@@ -14,26 +14,26 @@ use std::os::wasi::io::{FromRawFd, RawFd};
 mod veracruz_si_import {
     #[link(wasm_import_module = "veracruz_si")]
     extern "C" {
-        pub fn fd_create(x: u32) -> u32;
+        pub fn fd-create(x: u32) -> u32;
```

Comment:
I doubt this will compile because of syntax error. You may have to revert some change back to `fd_create`. Note that there are difference between file directory and project names and programing syntax.

Reply:
So, I need to ONLY change fd_create to fd-create when it is a 'directory name'??

Reply:
I think so.